FORMAT.md

```diff
@@ -38,6 +38,16 @@ The robot element is the root element of a robot model.
 - `trans` (translation vector) specify the translation to the base frame of the model; defaults to (0,0,0)
 - `description` (string) human readable description of this robot
 
+**Content:**
+Zero or more of the following:
+- `<actuator>`
+- `<bracket>`
+- `<link>`
+- `<rigid-body>`
+- `<joint>`
+- `<end-effector>`
+- `<include>`
+
```

Comment:
I went ahead and based this on other XML spec I've seen, where they show the allowable element children as "content"